Database/Corrections/Wotlk/wotlkItemFixes.lua

```diff
@@ -206,6 +206,10 @@ function QuestieWotlkItemFixes:Load()
         [41399] = {
             [itemKeys.npcDrops] = {},
         },
+        [42203] = {
+            [itemKeys.npcDrops] = {29380},
+            [itemKeys.startQuest] = 12979,
```

Comment:
Does defining `itemKeys.startQuest` fix the chat prompt ("Picked up X item which starts Y quest!")? I've noticed that none of the WotLK quest-start items print that text, and as far as I can see they're missing this `itemKey`.

Reply:
Yes, that is absolute true. Good catch @dscreet 💪🏻 I'll see if we can script these missing fields into the other quest starting items.

Reply:
Didn't know that was the case haha. I just double-checked all fields and saw this was missing. Gd to know.